### PR TITLE
[Finished] Documents Endpoints

### LINKED ISSUE #24: Create Backend Endpoints
Create Endpoints for Messages, Documents, and Other Databases.

### PR DESCRIPTION
**Pull request by uawasth2**
Resolves #24 for Documents only

<Brief description of the added features or other changes/updates (either bulletpointed list, small paragraph, etc.)>

### CODE REVIEW

**Comment by uawasth2** [comment]
All get endpoints for documents finished

**Comment by uawasth2** [comment]
Aight I have done everything but frontend has issues from the pull I made.

File: backend/tests/test_document.py [hunk 1]
```diff
@@ -1,44 +1,63 @@
...
+    assert ret_dict["result"]["documents"]["Pending"][0]["fileID"] == "DunDunDun"
+    assert ret_dict["result"]["documents"]["Pending"][0]["userID"] == "WompWomp"
+    assert ret_dict["result"]["documents"]["Pending"][0]["status"] == "Pending"
+
+    rs = client.get("/document?uid=WompWomp")
+    ret_dict = rs.json
+    assert len(ret_dict["result"]["documents"]) == 4
+    assert ret_dict["result"]["documents"]["Pending"][0]["fileID"] == "DunDunDun"
+    assert ret_dict["result"]["documents"]["Pending"][0]["userID"] == "WompWomp"
+    assert ret_dict["result"]["documents"]["Pending"][0]["status"] == "Pending"
```
Comment (rgychiu) [inline backend/tests/test_document.py]: Everything looks good, I would like to add another test case or two if possible utilizing your post endpoint

Comment (uawasth2) [reply backend/tests/test_document.py]: Done

**Pull request merged**